login/webpack/dist/app.js

```diff
@@ -0,0 +1 @@
.dropdown-menu {\n  font-size: 1.3rem !important;\n}\n.dropdown-item {\n  padding-top: 5px;\n  padding-bottom: 5px;\n}\n.dropdown-item:active {\n  background-color: transparent;\n}\n.dropdown-header {\n  font-size: 1.3rem !important;\n}\n@keyframes spin {\n  0% {\n    transform: rotate(0deg);\n  }\n  100% {\n    transform: rotate(360deg);\n  }\n}\n.loader {\n  border: 16px solid #f3f3f3;\n  border-top: 16px solid #00a3dd;\n  border-radius: 50%;\n  width: 120px;\n  height: 120px !important;\n  animation: spin 0.5s linear infinite;\n  margin: 0 auto;\n  margin-top: 20px;\n}\n.miniLoader {\n  border: 10px solid #f3f3f3;\n  border-top: 10px solid #00a3dd;\n  width: 30px;\n  height: 30px !important;\n  border-radius: 50%;\n  animation: spin 0.5s linear infinite;\n  margin: 0 auto;\n  margin-top: 10px;\n}\n.featured {\n  position: absolute;\n  text-align: center;\n  bottom: 0;\n  width: 100%;\n  font-weight: 600;\n  font-size: 16px;\n  background: #40b1f3;\n  color: #fff;\n  z-index: 1;\n}\n.container-3 {\n  width: 1070px;\n  padding-right: 15px;\n  padding-left: 15px;\n  margin-right: auto;\n  margin-left: auto;\n}\n@media (max-width: 767px) {\n  .container-3 {\n    width: 100%;\n    text-align: center;\n  }\n}\n@media (min-width: 768px) and (max-width: 1091px) {\n  .container-3 {\n    width: 100%;\n  }\n}\n.view-title {\n  padding-top: 20px;\n  padding-bottom: 20px;\n  font-weight: bold;\n}\n.ep-btn {\n  min-height: 40px;\n  min-width: 100px;\n  margin-right: 10px;\n  border: 0;\n  margin-top: 5px;\n  margin-bottom: 5px;\n  padding: 4px 8px;\n  font-weight: bold;\n  font-family: \'Montserrat\', sans-serif;\n  text-transform: uppercase;\n  outline: 0;\n  transition: 0.1s;\n  background-color: #fff;\n  border: 1px solid #40b1f3;\n  color: #40b1f3;\n  cursor: pointer;\n}\n.ep-btn:hover {\n  background-color: #40b1f3;\n  color: #fff;\n}\n.ep-btn:focus {\n  outline: 2px solid #007bff80;\n}\n.ep-btn:active {\n  background-color: #0062cc;\n}\n.ep-btn:disabled {\n  color: #b8e2fb;\n  border-color: #b8e2fb;\n  cursor: default;\n}\n.ep-btn:disabled:hover {\n  background-color: #fff;\n}\n.ep-btn:disabled:focus,\n.ep-btn:disabled:active {\n  outline: none;\n  background-color: #fff;\n}\n.ep-btn.primary {\n  border: 0;\n  color: #fff;\n  background-color: #40b1f3;\n}\n.ep-btn.primary:focus {\n  outline: 2px solid #007bff80;\n}\n.ep-btn.primary:active {\n  background-color: #0062cc;\n}\n.ep-btn.primary:disabled {\n  cursor: default;\n  background-color: #b8e2fb;\n  outline: 0;\n}\n.ep-btn.wide {\n  width: 280px;\n}\n@media (max-width: 768px) {\n  .ep-btn.wide {\n    width: 100%;\n  }\n}\n.ep-btn.small {\n  min-height: 30px;\n}\n',""])},,function(n,e,o){n.exports=o.p+"fonts/glyphicons-halflings-regular.woff"},function(n,e,o){n.exports=o.p+"fonts/glyphicons-halflings-regular.ttf"},function(n,e,o){n.exports=o.p+"fonts/fa-brands-400.woff2"},function(n,e,o){n.exports=o.p+"fonts/fa-brands-400.woff"},function(n,e,o){n.exports=o.p+"fonts/fa-brands-400.ttf"}],[[16,2,1]]]);
```

Comment:
Who owns this code? This is the minified version.

Reply:
Updated .gitignore to fix that